woocommerce/woocommerce-gutenberg-products-block
# Prevent wp.components loading on frontend

There were some frontend/base components using the `@wordpress/components` package. This PR fixes that to reduce the number of assets required on frontend.

Fixes #1899

### How to test the changes in this Pull Request:

1. Checkout this branch
2. Do a production build `npm run build`
3. Check network tools on the cart page to confirm `components` scripts are not loaded
4. Check /build/cart-frontend.asset.php for wp-components. It should not be there.
